test/streamparse/test_dsl.py

```diff
@@ -1,95 +1,68 @@
 import unittest
 import logging
 
-from pystorm.spout import Spout
-from pystorm.bolt import Bolt
-
-from streamparse.dsl.topology import Topology, Grouping, TopologyError
-from streamparse.dsl.spec import Spec
+from streamparse.dsl.exceptions import TopologyError
+from streamparse.dsl.stream import Grouping
+from streamparse.dsl.topology import Topology
+from streamparse.storm.bolt import Bolt
+from streamparse.storm.spout import Spout
```

Comment:
> At a minimum, Bolt and Spout should get aliased up to streamparse.storm

We already have the `streamparse.bolt.Bolt` and `streamparse.spout.Spout` aliases.  We certainly could just have those available as `streamparse.Bolt` and `streamparse.Spout` though.  It would probably make sense to do the same thing for `pystorm` as well.